config/dbconfig.txt

```diff
@@ -13,33 +13,30 @@ ADDRESS localhost
 PORT 3306
 
 ## Database for all SQL functions, not just feedback.
-FEEDBACK_DATABASE feedback
+FEEDBACK_DATABASE tgbrsql
 
 ## Prefix to be added to the name of every table, older databases will require this be set to erro_
 ## Note, this does not change the table names in the database, you will have to do that yourself.
 ##IE:
 ##	FEEDBACK_TABLEPREFIX
 ##	FEEDBACK_TABLEPREFIX SS13_
 ## Remove "SS13_" if you are using the standard schema file.
-FEEDBACK_TABLEPREFIX SS13_
+FEEDBACK_TABLEPREFIX
 
 ## Username/Login used to access the database.
-FEEDBACK_LOGIN username
+FEEDBACK_LOGIN root
```

Comment:
These values are used as the defaults for the standalone database package (https://tgstation13.download/database/) changing them would break those.

I would not be opposed to killing prefix and fixing our table names on prod. I now know it was a silly idea.

Also the database names and these `feedback` config names could be made better, I'd say something like tgstation would work better, but maybe those kinds of changes can happen in a different pr.

Reply:
this is just what i'm testing with, can't be bothered to set up a global gitignore, i'll change it back when it's ready